board/piksiv3/rootfs-overlay/etc/init.d/S75dev_boot

```diff
@@ -23,8 +23,8 @@ copy_from_sd() {
 copy_from_net() {
   local server_ip=$1
   local filename=$2
-  local serial_number=`cat /factory/serial_number`
-  local net_file="/PK$serial_number/$filename"
+  local uuid=`cat /factory/uuid`
+  local net_file="/PK$uuid/$filename"
```

Comment:
@jacobmcnamee this naming is a little funky:

`/PKa44a00e2-647a-420c-a7ed-f5975a202e48`

Do we need to preserve the `PK` prefix? Alternatives:

`/PK/a44a00e2-647a-420c-a7ed-f5975a202e48`
`/SN-a44a00e2-647a-420c-a7ed-f5975a202e48`
`/swift-nav/a44a00e2-647a-420c-a7ed-f5975a202e48`
`/swiftnav/a44a00e2-647a-420c-a7ed-f5975a202e48`

I don't have much of a horse in this race.

/cc @fnoble